students/ian_letourneau/mailroom/mailroomOO.py

```diff
@@ -0,0 +1,236 @@
+#!/usr/bin/env python3
+# Ian Letourneau
+# 5/11/2018
+# A script that stores values for donors and their donations.
+# The script can process requests from the user on various functions
+# to utilize these values
+import datetime
+
+# Build a dictionary of existing donors
+
+
+class Donor:
+    """An object that contains information for donors and their donations.
+        attribute name: donors name.
+        attribute donation_list: a list of given donations."""
+
+    def __init__(self, name, donations=None):
+        """Initialization for Donor objects.
+
+        Params name: Will be the name given to Donor object
+        Params donations: a value or list of values to be 
+            combined into donation_list attribute."""
+        self.name = name
+        if isinstance(donations, list):
+            self.donation_list = donations
+        elif donations == None:
```

Comment:
it is more pythonic to use `is` to test for None:
`elif donations is None` 

to simplify this:
```
if isinstance(donations, list):
    self.donation_list = donations
else:
    self.donations_list = [donations] if donations else []
```